dredd/api-description.yml

```diff
@@ -1479,110 +1479,476 @@ definitions:
+        properties:
+          general:
             type: object
             properties:
-              general:
+              randomizeProviders:
+                type: boolean
+              downloadPropers:
+                type: boolean
+              checkPropersInterval:
+                type: string
+              propersSearchDays:
+                type: integer
+              backlogDays:
+                type: integer
+              backlogFrequency:
+                type: integer
+              minBacklogFrequency:
+                type: integer
+              dailySearchFrequency:
+                type: integer
+              minDailySearchFrequency:
+                type: integer
+              removeFromClient:
+                type: boolean
+              torrentCheckerFrequency:
+                type: integer
+              minTorrentCheckerFrequency:
+                type: integer
+              usenetRetention:
+                type: integer
+              trackersList:
+                type: array
+              allowHighPriority:
+                type: boolean
+              useFailedDownloads:
+                type: boolean
+              deleteFailed:
+                type: boolean
+              cacheTrimming:
+                type: boolean
+              maxCacheAge:
+                type: integer
+          filters:
+            type: object
+            properties:
+              ignored:
+                type: array
+              undesired:
+                type: array
+              preferred:
+                type: array
+              required:
+                type: array
+              ignoredSubsList:
+                type: array
+              ignoreUnknownSubs:
+                type: boolean
+          showDownloadDir:
+            type: string
+          processAutomatically:
+            type: boolean
+          postponeIfSyncFiles:
+            type: boolean
+          postponeIfNoSubs:
+            type: boolean
+          renameEpisodes:
+            type: boolean
+          createMissingShowDirs:
+            type: boolean
+          addShowsWithoutDir:
+            type: boolean
+          moveAssociatedFiles:
+            type: boolean
+          nfoRename:
+            type: boolean
+          airdateEpisodes:
+            type: boolean
+          unpack:
+            type: boolean
+          deleteRarContent:
+            type: boolean
+          noDelete:
+            type: boolean
+          processMethod:
+            type: string
+          reflinkAvailable:
+            type: boolean
+          autoPostprocessorFrequency:
+            type: integer
+          syncFiles:
+            type: array
+          fileTimestampTimezone:
+            type: string
+          allowedExtensions:
+            type: array
+          extraScripts:
+            type: array
+          extraScriptsUrl:
+            type: string
+          multiEpStrings:
+            type: object
+      notifiers:
```

Comment:
@medariox @sharkykh the dredd tests is failing. I'm not sure how we should configure these sections. Can anyone double check this?

Reply:
What did you use to see the diff?

Reply:
Aah i missed that. I went straight into the expected error.